### ISSUE TITLE
"Your administrator requires a security scan of this item." Great, how do I scan it?

### ISSUE THREAD

**Issue by ryancampfire** [MicrosoftDocs/windows-itpro-docs#8112]
For some reason the "Your administrator requires a security scan of this item." pop up links to this page instead of a page that tells me how to scan the item. 

Apparently it will take up to 10 seconds to scan the item. That sounds fine. I'm on board. Let's scan the item. I would love to scan this item. How do I scan the item? 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9bd0fe29-5db5-b752-5bcf-5ac2581baa9a
* Version Independent ID: 52b8d4ca-54ea-7345-12b0-df4fdf633083
* Content: [Enable Block at First Sight to detect malware in seconds - Windows security](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-antivirus/configure-block-at-first-sight-microsoft-defender-antivirus)
* Content Source: [windows/security/threat-protection/microsoft-defender-antivirus/configure-block-at-first-sight-microsoft-defender-antivirus.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/threat-protection/microsoft-defender-antivirus/configure-block-at-first-sight-microsoft-defender-antivirus.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @denisebmsft
* Microsoft Alias: **deniseb**

**Comment by VLG17** [comment]
@ryancampfire As the referenced article points out, this happens because of the real time protection features of Windows Defender Antivirus. As stated in the article, you don't need to do something. Everything happens in the background and if indeed a malitious file is found it will be blocked and you'll be notified.

> **Block at first sight provides a way to detect and block new malware within seconds**. This protection is enabled by default when certain prerequisite settings are also enabled. In most cases, these prerequisite settings are also enabled by default, **so the feature is running without any intervention**.

If it's a corporate machine you could contact your Administrator and ask for help with setting exclusions for what you're using and know it's safe so it does not get scanned.

If it's a personal machine, you can set these exclusions on your own.

**Comment by e0i** [comment]
The issue is being closed since your question seems to be answered. Feel free to re-open this issue if you feel that it hasn't been answered or that there are further suggestions to improve the documentation itself.

Thanks.

**Comment by RAJU2529** [comment]
@e0i  I have pending PR, so verify all

**Comment by illfated** [comment]
@RAJU2529 : Do you have a link to your PR?

By the way, when you say "so verify all", could you rephrase or provide more details?

**Comment by RAJU2529** [comment]
@illfated . https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8356

**Comment by illfated** [comment]
Thank you for the links. If I may give you some advice about posting, I recommend refraining from posting in tickets unrelated to the ones you have questions about.

If you want to request attention to your Pull Requests or tickets, please post such requests in the same ticket or PR as the one where the attention is wanted.
